packages/esm-patient-orders-app/src/order-basket/order-basket.workspace.tsx

```diff
@@ -110,8 +118,11 @@ const OrderBasket: React.FC<DefaultPatientWorkspaceProps> = ({
     closeWorkspace({ onWorkspaceClose: clearOrders });
   }, [clearOrders, closeWorkspace]);
 
+  const isRdeEnabled = useFeatureFlag('rde');
+
   return (
     <>
+      {isRdeEnabled && <ExtensionSlot name="visit-context-header-slot" state={{ patientUuid }} />}
```

Comment:
This is what I am talking about, why should we go manually into every patient chart workspace and render the extension slot, when we can use the workspace system to do the same?